packages/trader/src/Modules/Reports/Containers/statement.jsx

```diff
@@ -93,6 +93,8 @@ const Statement = ({
         return map;
     }, {});
 
+    const desc_titles = [localize('Address:'), localize('Transaction:')];
```

Comment:
okay I see it's from our own codebase. then we should have id to identify them and not parse the string, as it won't work for non-en.

Reply:
The response string will look like this: `Address: 0x54d78192a55BE74a9cG5751303845F96707050E1, transaction: 0x17249f7a995e1a917013b1d79590630cbc7ab0b1bb972c499327df8c554d90fa` but the translation is happening just for the display text. Can I still parse it if it's directly from BE or is there a better way to ID it?